[WIP] Updated the local installation file (#388)
actualbudget/docs: docs/install/local.md

# Local Installation

> The easiest way to get the Actual running on locally is to use the [actual-server](https://github.com/actualbudget/actual-server). 

Reviewer: Fixing this during cleanup of abandon PR

Actual Server is the server element of the Actual which is used for syncing changes across devices and which comes with the latest version of [Actual Web.](https://github.com/actualbudget/actual)

## Prerequisites

> - The Actual Server requires Node.js v18 and greater. You can download and install latest version of the Node.js from [Node.js website](https://nodejs.org/en/download) (we recommend downloading the “LTS” version). 

Reviewer: Fixed during cleanup of abandoned PR.

- Consider using a tool like [nvm](https://github.com/nvm-sh/nvm) or [asdf](https://asdf-vm.com) to install and manage multiple versions of Node.js.
- You’ll also need to have Git installed. The Git website has [instructions for downloading and working with Git for all supported operating systems](https://git-scm.com/download).
- Actual uses yarn packages. You can install [yarn](https://yarnpkg.com/getting-started/install) using the following command:

  ```bash
  npm install --global yarn
  ```

## Installing Actual

1. After the prerequisites are fulfilled, clone the [Actual Server](https://github.com/actualbudget/actual-server) project in your project root directory where you want to install Actual.
  ```bash
  git clone https://github.com/actualbudget/actual-server.git
  ```

2. Navigate to the Actual Server in your project root director.
    ```bash
    cd actual-server
    ```
3. Install all the required dependencies using yarn.
    ```bash
    yarn install
    ```

## Running Actual

After the Actual server is installed, start the Actual server by running the following command:
```bash
yarn start
```

## Accessing Actual

> After the Actual server is started, you can access the Actual server from local server logged in the CLI as  [http://localhost:5006](http://localhost:5006). 

Reviewer: I agree with @Teprifer - but I also like some of the proposed changes. 

What about _After the servers have started, you can access Actual using your browser at [http://localhost:5006](http://localhost:5006). _

This is a bit more to the point compared to the original text and maybe a less confusing that the proposed change.

When accessing Actual for the first time, you may be prompted to provide a URL for the server. For a local installation, click the **Use localhost:5006** button to use the local server to access Actual.

## Updating Actual

Actual is constantly evolving to include new features and improve user's experience. It is recommended to update your local installation with our latest releases.

> To update with our latest releases, use `git pull` command from the `master` branch of your local directory of Actual. After updating the changes, execute the `yarn install` to download the latest updates from the releases.

Reviewer: I agree with @Teprifer - even if the proposed change is shorter, there are details that only seasoned people will understand, which is missing. We always need to remember that this part of the documentation is not necessarily for the "experts".

> To update with our latest releases, use `git pull` command from the `master` branch of your local directory of Actual. After updating the changes, execute the `yarn install` to download the latest updates from the releases.

Author: Got it @RubenOlsen

# Local Installation

The easiest way to get Actual running locally is to use [actual-server](https://github.com/actualbudget/actual-server). 

> Actual Server is the server element of Actual that is used for syncing changes across devices. It comes with the latest version of [Actual Web.](https://github.com/actualbudget/actual)

Reviewer 2: ```suggestion
Actual server is used for syncing changes across devices. It comes with the latest version of the [Actual web client](https://github.com/actualbudget/actual).
```

## Prerequisites

> - The Actual Server requires Node.js v18 or greater. You can download and install latest version of the Node.js from [Node.js website](https://nodejs.org/en/download) (we recommend downloading the “LTS” version). 

Reviewer 2: ```suggestion
- The Actual server requires Node.js v18 or greater. You can download and install the latest version of Node.js from [Node.js website](https://nodejs.org/en/download) (we recommend downloading the “LTS” version). 
```

- Consider using a tool like [nvm](https://github.com/nvm-sh/nvm) or [asdf](https://asdf-vm.com) to install and manage multiple versions of Node.js.
- You’ll also need to have Git installed. The Git website has [instructions for downloading and working with Git for all supported operating systems](https://git-scm.com/download).
- Actual uses yarn packages. You can install [yarn](https://yarnpkg.com/getting-started/install) using the following command:

  ```bash
  npm install --global yarn
  ```

## Installing Actual

> 1. After the prerequisites are fulfilled, clone the [Actual Server](https://github.com/actualbudget/actual-server) project in your project root directory where you want to install Actual.

Reviewer 2: ```suggestion
1. After the prerequisites are fulfilled, clone the [Actual server](https://github.com/actualbudget/actual-server) project in your project root directory where you want to install Actual.
```

  ```bash
  git clone https://github.com/actualbudget/actual-server.git
  ```

> 2. Navigate to the Actual Server in your project root director.

Reviewer 2: ```suggestion
2. Navigate to the Actual server in your project root director.
```

    ```bash
    cd actual-server
    ```

> 3. Install all the required dependencies using yarn.

Reviewer 2: ```suggestion
3. Install all the required dependencies using `yarn`.
```

    ```bash
    yarn install
    ```

## Running Actual

After the Actual server is installed, start the Actual server by running the following command:
```bash
yarn start
```
Note that if you restart your computer, you’ll have to run this command again to start the server.

## Accessing Actual

After the server has been started, you can access Actual using your browser at [http://localhost:5006](http://localhost:5006). 

When accessing Actual for the first time, you may be prompted to provide a URL for the server. For a local installation, click the **Use localhost:5006** button to use the server you've [configured](https://actualbudget.org/docs/config/).

## Updating Actual

> 1. Stop the server if it’s running. You can use the keyboard shortcut <kbd>"ctrl" </kbd> <kbd>"c" </kbd> (even on macOS) to stop the server, or close the terminal window it’s running from.

Reviewer 2: ```suggestion
1. Stop the server if it’s running. You can use the keyboard shortcut <kbd>CTRL-C</kbd> (even on macOS) to stop the server or close the terminal window it’s running from.
```

2. Run `git pull` from the directory you cloned the project into. This will download the latest server code.
3. Run `yarn install` from that same directory. This will download the latest web client code, along with any updated dependencies for the server.
4. Restart the server by running `yarn start`.

> Actual is constantly evolving to include new features and improve user's experience. It is always recommended to update your local installation with our [latest releases](https://actualbudget.org/docs/releases).

Reviewer 2: ```suggestion
Actual is constantly evolving to include new features and improve the user's experience. It is always recommended that your local installation be updated with our [latest releases](https://actualbudget.org/docs/releases).
```

```bash
yarn start
```
Note that if you restart your computer, you’ll have to run this command again to start the server.

## Accessing Actual

> After servers have started, you can access Actual using your browser at [http://localhost:5006](http://localhost:5006). 

Reviewer 3: Slight tweak, should be 'server' singular, and has vs have, e.g.

After the server has been started
